lms/djangoapps/discussion/django_comment_client/tests/test_utils.py

```diff
@@ -1151,38 +1152,41 @@ def test_beta_user(self):
         Verify that the beta user can access the beta and global
         discussion topics.
         """
-        self.assert_category_map_equals(
-            {
-                'subcategories': {
-                    'Week 1': {
-                        'subcategories': {},
-                        'children': [
-                            ('Visible to Beta', 'entry'),
-                            ('Visible to Everyone', 'entry')
-                        ],
-                        'entries': {
-                            'Visible to Beta': {
-                                'sort_key': None,
-                                'is_divided': False,
-                                'id': 'beta_group_discussion'
-                            },
-                            'Visible to Everyone': {
-                                'sort_key': None,
-                                'is_divided': False,
-                                'id': 'global_group_discussion'
-                            }
+
+        expected = {
+            'subcategories': {
+                'Week 1': {
+                    'subcategories': {},
+                    'children': sorted([
```

Comment:
In python3 the expected results order appear differently and causing the failures.